ddp-workspace/projects/ddp-prion/src/assets/i18n/en.json

```diff
@@ -142,8 +142,23 @@
       }
     },
     "ThirdParty": {
-      "Title": "",
-      "Description": "For example, we use Google Cloud to host the Registry and Auth0 as an authentication system to provide access to the Registry."
+      "Title": "List of Third-Party Tools",
+      "Description": "The Prion Registry leverages the following third-party tools that may receive information about a user:",
+      "Tools": {
+        "Sendgrid": "Sendgrid",
+        "Auth0": "Auth0",
+        "Tcell": "Tcell",
+        "Google_Cloud": "Google Cloud",
+        "Google_Analytics": "Google Analytics"
+      },
+      "Usage": {
+        "email": "as an email service",
+        "authentication": "as an authentication system to provide access to the Registry",
+        "security": "as an Registry cloud firewall and runtime protection",
```

Comment:
Typo here: "as an Registry" should be "as a Registry"

Reply:
Done in 13fa563